lib/app/utils/oauth2_error_messages.rb

```diff
@@ -7,7 +7,11 @@ def no_instance_for_state_error_message
         %(
           <p>
             Inferno has detected an issue with the SMART launch.
-            No actively running launch sequences found with a state of #{params[:state]}.
+            <% if params[:state].nil?} %>
```

Comment:
These ERB tags are coming through in the HTML output, because (I think) this is a raw string.
<img width="1275" alt="Screen Shot 2020-06-25 at 4 11 20 PM" src="https://user-images.githubusercontent.com/49001/85790691-a6867700-b6fe-11ea-9a76-ab53c7acd952.png">

Reply:
Fixed in the last commit.  Great catch.

Reply:
Yup, looks good.